docs/plugins/builtin/data-sources/inline.md

```diff
@@ -0,0 +1,26 @@
+---
+title: inline
+plugin:
+  name: blackstork/builtin
+  description: ""
+  tags: []
+  version: "v0.0.0-dev"
+  source_github: "https://github.com/blackstork-io/fabric/tree/main/internal/builtin/"
+resource:
+  type: data-source
+type: docs
+---
+
+{{< breadcrumbs 2 >}}
+
+{{< plugin-resource-header "blackstork/builtin" "builtin" "v0.0.0-dev" "inline" "data source" >}}
+
+The data source is built-in, which means it's a part of `fabric` binary. It's available out-of-the-box, no installation required.
+
+## Configuration
+
+The data source doesn't support configuration.
+
+## Usage
+
+The data source doesn't define any execution parameters.
```

Comment:
I think this one is hard to generate based on `.Args` specs because it is `nil` and any parameters can be set. Maybe hardcode example of usage with `if` condition in that case?

Reply:
My plan was to leaving it as is until https://github.com/blackstork-io/fabric/issues/124 is ready